sso-kit-starter/src/main/java/com/vaadin/sso/starter/SingleSignOnConfiguration.java

```diff
@@ -49,7 +48,6 @@
 @AutoConfiguration
 @EnableWebSecurity
 @Conditional(ClientsConfiguredCondition.class)
-@ConditionalOnProperty(name = "auto-configure", prefix = SingleSignOnProperties.PREFIX, matchIfMissing = true)
```

Comment:
The javadoc for this class should be updated also, see line[41](https://github.com/vaadin/sso-kit/pull/87/files#diff-979e79b39d14f899a14a748ef745052d38254bd25da99d3ccd0aaee5068754cbR41).

For the other `SingleSignOnConfiguration` class in the `sso-kit-starter-hilla` there no javadoc yet. Could we add one similar to this one?